indexer/osm_editor.cpp

```diff
@@ -556,6 +556,21 @@ bool Editor::GetEditedFeatureStreet(FeatureType const & feature, string & outFea
   return true;
 }
 
+vector<uint32_t> Editor::GetFeaturesByStatus(MwmSet::MwmId const & mwmId, FeatureStatus status) const
```

Comment:
How about to implement a test for that? Take a look at ours search_integration_tests, especially at sesrch_query_v2 test - it creates a couple of test mwm and performs simple search on it. Try to create a couple of test features through editor and try to find them.